soc/arm/microchip_mec/mec1501/device_power.c

```diff
@@ -194,23 +191,128 @@ static void deep_sleep_restore_timers(void)
 
 	p = &ds_timer_tbl[0];
 	for (i = 0; i < NUM_DS_TIMER_ENTRIES; i++) {
-		REG32(p->addr) = timers[i] & ~p->restore_mask;
+		REG32(p->addr) = ds_ctx.timers[i] & ~p->restore_mask;
 		p++;
 	}
 }
 
+#ifdef DEEP_SLEEP_PERIPH_SAVE_RESTORE_EXTENDED
+
+static void deep_sleep_save_blocks(void)
+{
+	/* ADC Power saving feature is enabled */
+	ds_ctx.adc_info.adc_ctrl = ADC_REGS->CONTROL;
+	ds_ctx.adc_info.adc_cfg = ADC_REGS->CONFIG;
+	ADC_REGS->CONTROL = 0;
+	ADC_REGS->CONFIG = 0;
+
+	ds_ctx.peci_info.peci_ctrl = PECI_REGS->CONTROL;
+	ds_ctx.peci_info.peci_dis = ECS_REGS->PECI_DIS;
+	PECI_REGS->CONTROL = 0x08;
```

Comment:
instead of  magic number, MCHP_PECI_CTRL_PD and MCHP_PECI_CTRL_RST can be used.

Reply:
sure